risc0/core/src/field/mod.rs

```diff
@@ -108,12 +108,6 @@ pub trait Elem: 'static
     /// methods, this may be called on an INVALID element.
     fn is_valid(&self) -> bool;
 
-    /// Returns true if this element is represented in reduced/normalized form.
-    /// Every element has exactly one reduced form. For a field of prime order
-    /// P, this typically means the underlying data is < P, and for an extension
-    /// field, this typically means every component is in reduced form.
-    fn is_reduced(&self) -> bool;
```

Comment:
This breaks semver; we could leave it stubbed out with an impl that always returns the appropriate value.